loadtest/main.go

```diff
@@ -100,6 +101,42 @@ func runOnce(config Config) {
 	go client.ValidateTxs()
 }
 
+// TODO: runContinuously continously sends transactions without closing and reopening channel
+// Ran into seq acc mismatch issues
+// func runContinously(config Config) {
```

Comment:
`runContinuously` would send tx's without closing the channel. Included a `prevSeqDelta` that would count the number of rounds accounts were involved in so that we can increment seq number properly. However still ran into acc seq issues particularly the expected acc seq number was 1 larger than the actual acc seq number. Will investigate as a P1 and add. For now, we resort to just recreating the client

Reply:
Makes sense